fh-messaging/lib/services/mbaas_metrics.js

```diff
@@ -210,13 +210,35 @@ module.exports = function(mongodb) {
           q[opt] = mReduceConf[opt];
         }
       }
-
-      var options = {query : q, out: {reduce: metric, db: dbName}, scope: {ts: from}};
-      logger.info("Rollup options = " + JSON.stringify(options));
-      collection.mapReduce(mReduceConf.map, mReduceConf.reduce, options, function(err,results) {
-        var filteredErr = helpers.ignoreSomeErrs(err);
-        return callback(filteredErr, results);
+      mongodb.collection('apprequestsdest', function(err, collection) {
+        collection.find({}, function(err, cursor) {
+          cursor.toArray(function(err, docs) {
+            console.log('@@@@@appprequestdest doooocs', docs);
+            var options = {query : q, out: {reduce: metric, db: dbName}, scope: {ts: from}};
+            logger.info("Rollup options = " + JSON.stringify(options));
+            collection.mapReduce(mReduceConf.map, mReduceConf.reduce, options, function(err,results) {
+              var filteredErr = helpers.ignoreSomeErrs(err);
+              collection.find({}, function(err, cursor) {
+                cursor.toArray(function(err, docs) {
+                  console.log('@@@@@mbaas_appprequestdest doooocs', docs);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)